test/rng_testsuite/extra_methods_tests/engines_methods.pass.cpp

```diff
@@ -0,0 +1,256 @@
+// -*- C++ -*-
+//===-- engines_methods.pass.cpp ------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+//
+// Abstract:
+//
+// Testing of different engines' methods
+
+#include "support/utils.h"
+
+#if TEST_DPCPP_BACKEND_PRESENT
+#    include <iostream>
+#    include <vector>
+#    include <CL/sycl.hpp>
+#    include <oneapi/dpl/random>
+
+const auto SEED = 777;
+const auto N_GEN = 960;
+const auto MINSTD_A = 48271;
+const auto MINSTD_C = 0;
+const auto MINSTD_M = 2147483647;
+const auto MINSTD_MIN = 1;
+const auto MINSTD_MAX = 2147483646;
+const auto RANLUX24_BASE_W = 24;
+const auto RANLUX24_BASE_S = 10;
+const auto RANLUX24_BASE_R = 24;
+const auto RANLUX24_BASE_MIN = 0;
+const auto RANLUX24_BASE_MAX = 16777215;
+const auto RANLUX24_P = 223;
+const auto RANLUX24_R = 23;

```

Comment:
I think it's better to mark it as `constexpr`, not `const` to make it available at compile-time. Some of them are used in kernel. So, `constexpr` will help JIT Compiler generate more optimized code, and it potentially can reduce execution time of test.

Reply:
changed to constexpr